.gitignore

```diff
@@ -0,0 +1,6 @@
+*.egg-info
```

Comment:
Cool. Wasn't sure if it was standard practice to keep those in global gitignore.